recipes/ostaphep/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "OstapHEP" %}
+{% set version = "1.3.3.0" %}
+{% set hash = "14447989be2ef14d45db292c41db8b18eaf07797ba4812d2775d75835ba90515" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OstapHEP/ostap/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+  folder: ostap
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - root >=6.16.0
+    - python
+    - ipython
+  run:
+    - python 
+    - root 
+    - future
+    - ipython
+test:
+  imports:
+    - ostap
+    - ostap.core.core
+about:
+  home: https://github.com/OstapHEP/ostap
+  license: BSD 3-Clause 
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: BSD-3-Clause 
```